rclcpp/include/rclcpp/client.hpp

```diff
@@ -42,32 +46,28 @@ class ClientBase
 
   RCLCPP_PUBLIC
   ClientBase(
-    std::shared_ptr<rmw_node_t> node_handle,
-    rmw_client_t * client_handle,
+    std::shared_ptr<rcl_node_t> node_handle,
     const std::string & service_name);
 
   RCLCPP_PUBLIC
-  virtual ~ClientBase();
```

Comment:
I'm not sure. Previously the child templated class `Client` did not implement a custom destructor, and I add one in this pull request. Maybe I needed to delete this declaration to make it work. I can put this back in and see.